src/api/index.js

```diff
@@ -1,5 +1,8 @@
 import axios from 'axios';
 
+//export const BASE_URL = 'https://fast-savannah-33549.herokuapp.com/api';
+export const BASE_URL = 'http://localhost:5000/api';
```

Comment:
We can use this for now, but we will eventually want to move over to using the heroku data once that layer has settled down.